src/main/java/uk/ac/sanger/sccp/stan/config/StanConfig.java

```diff
@@ -0,0 +1,18 @@
+package uk.ac.sanger.sccp.stan.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Stan main config
+ * @author dr6
+ */
+@Configuration
+public class StanConfig {
+    @Value("${stan.root}")
+    String root;
+
+    public String getRoot() {
+        return this.root;
+    }
+}
```

Comment:
Would you please add the description (you have given me) here?

Reply:
The root (e.g. "https://stan.psd.sanger.ac.uk/") is used to construct the download url for the release file.

Reply:
Sorry, I mean If you can give this as a comment in code, it would be great

Reply:
OK. Added a comment.